pages/settings/wallets/index.js

```diff
@@ -27,12 +29,18 @@ export default function Wallet ({ ssrData }) {
       <div className='py-5 w-100'>
         <h2 className='mb-2 text-center'>attach wallets</h2>
         <h6 className='text-muted text-center'>attach wallets to supplement your SN wallet</h6>
+        <div className='text-center'>
+          <Link href='/wallet/logs' className='text-muted fw-bold text-underline'>
+            wallet logs
+          </Link>
+        </div>
```

Comment:
I moved the attached wallet logs link to the attached wallet page because it didn't feel like it belonged on the `/wallet` page.

As a thought experiment, we could put every button on the site on the homepage and everything is "one click" away. That's obviously silly, so imho "one click" is not something we should optimize for singularly.